pkg/apis/core/types.go

```diff
@@ -417,19 +417,15 @@ type PersistentVolumeClaimSpec struct {
 	// Value of Filesystem is implied when not included in claim spec.
 	// +optional
 	VolumeMode *PersistentVolumeMode
-	// This field can be used to specify that the volume should initially contain
-	// data rather than being empty (the default).
-	// If the VolumeSnapshotDataSource feature gate is enabled, the object may be
-	// an existing VolumeSnapshot object (snapshot.storage.k8s.io/VolumeSnapshot)
-	// in the same namespace.
-	// If the VolumePVCDataSource feature gate is enabled, the object may be an
-	// existing PVC (PersistentVolumeClaim) in the same namespace.
-	// If the AnyVolumeDataSource feature gate is enabled, the object may be any
-	// existing object in the same namespace.
-	// If the provisioner can support the specified data source, it will create
-	// a new volume based on the contents of the specified PVC, Snapshot, or custom
-	// resource.
-	// If the provisioner does not support the specified data source, the volume will
+	// This field can be used to specify either:
+	// * An existing VolumeSnapshot object (snapshot.storage.k8s.io/VolumeSnapshot - Beta)
+	// * An existing PVC (PersistentVolumeClaim)
```

Comment:
@j-griffith do you plan on updating alpha-beta designation for cloning in your PR?

Reply:
I see cloning is GA now.